packages/stylelint-plugin-dialtone/package.json

```diff
@@ -50,7 +51,13 @@
     "test": "node --test ./tests/lib/rules/*.mjs"
   },
   "devDependencies": {
-    "stylelint-test-rule-node": "^0.2.1"
+    "@eslint/js": "^9.31.0",
+    "eslint": "^9.31.0",
+    "eslint-plugin-eslint-plugin": "^6.3.1",
+    "eslint-plugin-n": "^17.0.0",
+    "globals": "^16.3.0",
+    "stylelint-test-rule-node": "^0.2.1",
+    "vue-eslint-parser": "^10.0.0"
```

Comment:
This were missing dependencies?

Reply:
Some are only for this package, but there are also a few duplicates. Fixed.